src/js/events/ChronosActions.js

```diff
@@ -17,11 +18,17 @@ const ChronosActions = {
           url: `${Config.rootUrl}/chronos/jobs`,
           data: [{name: 'embed', value: 'activeJobs'}],
           success: function (response) {
-            AppDispatcher.handleServerAction({
-              type: REQUEST_CHRONOS_JOBS_SUCCESS,
-              data: response
-            });
-            resolve();
+            let data;
+            try {
+              data = ChronosUtil.parseJobs(response);
+              AppDispatcher.handleServerAction({
+                type: REQUEST_CHRONOS_JOBS_SUCCESS,
+                data
+              });
+              resolve();
+            } catch (error) {
+              this.error(error);
+            }
```

Comment:
Becuase `ChronosUtil.parseJobs` can throw an error if the data being parsed is not of correct format. That is we in the `cacth` block call `this.error` with the message provided